### DISCUSSION TITLE
Welcome to pyprobables Discussions!

### DISCUSSION THREAD

**Discussion by barrust** [barrust/pyprobables#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hfnhf** [comment]
Hello, I'm a hobby programmer but I work with huge amounts of sensor and measurement data in a manufacturing corporation (mostly with domain knowledge of the process and premade statistical tools)
